munigeo/importer/turku.py

```
@@ -0,0 +1,641 @@
+import requests
+import yaml
+from django import db
+from django.contrib.gis import gdal
+from django.contrib.gis.gdal import CoordTransform, DataSource, SpatialReference
+from django.contrib.gis.gdal.error import GDALException
+from django.contrib.gis.geos import GEOSGeometry, MultiPolygon, Point
+
+from munigeo import ocd
+from munigeo.importer.base import Importer, register_importer
+from munigeo.importer.sync import ModelSyncher
+from munigeo.models import *
```

Ruff (0.15.15)

[error] `from munigeo.models import *` used; unable to detect undefined names

(F403)